tests/test_api.py

```diff
@@ -55,3 +55,18 @@ async def test_wrong_api_response():
     api = cro.ApiProvider(auth_required=False)
     with pytest.raises(cro.ApiError):
         await api.post('account')
+
+
+# @pytest.mark.asyncio
+# async def test_api_rate_limits():
```

Comment:
probably you can create some Mock for aiohttp request session to track how much requests was fired